packages/react-vapor/src/components/browserPreview/tests/BrowserPreviewEmpty.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import {mount, ReactWrapper, shallow} from 'enzyme';
+import * as React from 'react';
+
+import {Svg} from '../../svg';
+import {BrowserPreviewEmpty, BrowserPreviewEmptyProps} from '../BrowserPreviewEmpty';
+
+describe('BrowserPreviewEmpty', () => {
+    let component: ReactWrapper<BrowserPreviewEmptyProps>;
+    const defaultProps: BrowserPreviewEmptyProps = {
+        onClick: jest.fn(),
+    };
+
+    const mountWithProps = (props: BrowserPreviewEmptyProps, children?: React.ReactNode) => {
+        component = mount(<BrowserPreviewEmpty {...props}>{children}</BrowserPreviewEmpty>, {
+            attachTo: document.getElementById('App'),
+        });
```

Comment:
You don't need to attach your component to the DOM
```suggestion
        component = mount(<BrowserPreviewEmpty {...props}>{children}</BrowserPreviewEmpty>);
```